src/openforms/submissions/api/serializers.py

```diff
@@ -276,19 +276,27 @@ class SubmissionSuspensionSerializer(serializers.ModelSerializer):
             "The email address where the 'magic' resume link should be sent to"
         ),
     )
+    suspend_allowed = serializers.BooleanField()
```

Comment:
this isn't right - this is not a field that the client may control. It needs to be looked up from the serializer instance for validation via `self.instance.form.suspend_allowed`